samples/seat_massager/common/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub struct TargetedPayload {
+    pub instance_id: String,
+    pub member_path: String,
```

Comment:
Maybe add a short description for each struct field. What is the expected value for the `member_path` field?

Reply:
Fixed.